apps/preview/next/pages/showcases/Drawer/TransitionAndCloseButton.tsx

```diff
@@ -1,70 +1,105 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import { ShowcasePageLayout } from '../../showcases';
 // #region source
-import { SfDrawer, SfButton, SfIconFavorite, SfIconClose, useTrapFocus } from '@storefront-ui/react';
+import {
+  SfDrawer,
+  SfButton,
+  SfRadio,
+  SfIconFavorite,
+  SfIconClose,
+  useTrapFocus,
+  SfDrawerProps,
+} from '@storefront-ui/react';
 import { useState, useRef } from 'react';
-import { CSSTransition } from 'react-transition-group';
+import { Transition } from 'react-transition-group';
+import classNames from 'classnames';
 
 export default function DrawerWithTransition() {
   const [open, setOpen] = useState(false);
+  const [checkedPlacement, setCheckedPlacement] = useState<SfDrawerProps['placement']>('left');
   const nodeRef = useRef(null);
   const drawerRef = useRef(null);
 
   useTrapFocus(drawerRef, { activeState: open });
 
   return (
     <>
+      <fieldset>
+        <label className="flex items-center my-4 cursor-pointer">
+          <SfRadio
+            name="placement"
+            value="left"
+            checked={checkedPlacement === 'left'}
+            onChange={(event) => {
+              setCheckedPlacement(event.target.value as SfDrawerProps['placement']);
+            }}
+          />
+          <span className="ml-2">Slide from left</span>
+        </label>
+        <label className="flex items-center my-4 cursor-pointer">
+          <SfRadio
+            name="placement"
+            value="right"
+            checked={checkedPlacement === 'right'}
+            onChange={(event) => {
+              setCheckedPlacement(event.target.value as SfDrawerProps['placement']);
+            }}
+          />
+          <span className="ml-2">Slide from right</span>
+        </label>
```

Comment:
I think we could make it an iterable options.

```suggestion
      {options.map(({ label, value }) => (
        <label className="flex items-center my-4 cursor-pointer">
          <SfRadio
            name="placement"
            value={value}
            checked={checkedPlacement === value}
            onChange={(event) => {
              setCheckedPlacement(event.target.value as SfDrawerProps['placement']);
            }}
          />
          <span className="ml-2">{label}</span>
        </label>
      ))}
```

... and somewhere before define

```ts
const options = [
  { label: 'Slide from left', value: 'left' },
  { label: 'Slide from right', value: 'right' },
]
```